src/grisbi_osx.m

```diff
@@ -58,19 +58,28 @@
     return program_name;
 }
 
-static char const *get_program_dir()
+static gchar *get_program_dir()
 {
-    return g_path_get_dirname(get_program_name());
+    char *program_name = get_program_name();
+    gchar *program_dir = g_path_get_dirname(program_name);
+    free(program_name);
+    return program_name;
```

Comment:
I am not sure that returning `program_name` that was `free()` just above is a good idea.
Maybe you mean `return program_dir;`?

Reply:
You're right! :)
FIXED in last update